techLogger.js

```diff
@@ -1,20 +1,22 @@
 /**
  * Our Logger that wrap Winston logging library.
  */
-var path = require("path");
-var winston = require("winston");
-var splash = require("./lib/splash");
-var _ = require("lodash");
-
-var configurationDate;
-var consologger;
-var filogger;
-var syslogger;
-var syslogOptions;
+const path = require("path");
```

Comment:
{jlescotMode} select lines 4-9 and press F9* {/jlescotMode}

*Totally optionnal but I got used to it now :p